packages/react-dropdown/src/index.md

```diff
@@ -192,3 +192,22 @@ const items = [
   )}
 </Dropdown>;
 ```
+
+#### Dropdown with custom DropdownList style
+
```

Comment:
```suggestion
Depending by your use case, you may want to override the dropdown styles, for instance, to increase its width.

This can be achieved by using [Emotion's "component as selector"](https://emotion.sh/docs/styled#targeting-another-emotion-component) feature:
```